lib/engine/game/g_1880/game.rb

```diff
@@ -338,9 +340,14 @@ def next_round!
           @round =
             case @round
             when Engine::Round::Stock
-              @operating_rounds = @phase.operating_rounds
-              reorder_players
-              new_operating_round
+              if @saved_or_round
+                @log << '--Return to Operating Round--'
```

Comment:
I think its a new OR, not a return. The new OR just happens to begin where the last left off.

Reply:
I was expecting 1880 would detect the train event, set some state in the game, and override operating.finished? to true.